src/main/xerus/monstercat/downloader/Download.kt

```diff
@@ -81,10 +81,15 @@ class ReleaseDownload(private val release: Release, private var tracks: Collecti
 	
 	fun downloadTrack(releaseId: String, trackId: String, path: Path) {
 		val connection = APIConnection("api", "release", releaseId, "download").addQueries("method=download", "type=${QUALITY()}", "track=$trackId")
-		val entity = connection.getResponse().entity
+		val httpResponse = connection.getResponse()
+		val entity = httpResponse.entity
 		val contentLength = entity.contentLength
 		if(contentLength == 0L)
 			throw EmptyResponseException(connection.uri.toString())
+		if(!listOf("application/octet-stream", "audio/flac", "audio/wav", "audio/mpeg").contains(entity.contentType.value))
```

Comment:
I have a problem with this list - what if they offer ogg or some other format in the future? I'd rather blacklist than whitelist.
Or simply check if it starts with some sort of multimedia-prefix.

Reply:
I'll add the ``audio`` prefix

Reply:
630987bcf7bbd3769fbd774023501f9d272df537 is let ok or is run better ?

Reply:
unless there is a significant advantage, you should always prefer to have the receiver as parameter rather than receiver, so `let` over `run`. So that's correct 👍